internal/system/agent/services.go

```diff
@@ -131,6 +136,49 @@ func getMetrics(services []string, ctx context.Context) (MetricsRespMap, error)
 	return m, nil
 }
 
+func getHealth(services []string, ctx context.Context) HealthRespMap {
```

Comment:
Instead of returning a `HealthRespMap` struct alone from this func, return _along_ with it a var of type `error`.